lib/views/create_project_form.dart

```diff
@@ -33,12 +35,18 @@ class _CreateProjectFormState extends State<CreateProjectForm> {
         crossAxisAlignment: CrossAxisAlignment.end,
         children: [
           fields(context),
-          LoadingButton(
-            text: "Create my project",
-            loading: isLoading,
-            onPressed: () {
-              submit();
-            },
+          LenraFlex(
+            spacing: 16,
+            children: [
+              cancelButton(userHasApp),
```

Comment:
You should pass the context to the function instead of the `userHasApp` and create the `userHasApp` in `cancelButton`:

```suggestion
              cancelButton(context),
```